MapboxNavigation/NavigationViewController.swift

```diff
@@ -561,6 +561,14 @@ extension NavigationViewController: RouteControllerDelegate {
         }
         return advancesToNextLeg
     }
+    
+    @objc public func didEnterTunnel(_ darkModeEnabled: Bool) {
```

Comment:
`darkModeEnabled` makes it seem like it _was already_ enabled. I think it also might make sense to remove all notions of UI from this function since MapboxCoreNavigation deals non-UI related functionality.